Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -60,6 +60,7 @@
     'AP_ICEngine',
     'AP_Frsky_Telem',
     'AP_FlashStorage',
+    'AP_UAVCAN',
```

Comment:
This list isn't about what you use in runtime, it is about all the common libraries needed to be built in all vehicles and boards. When we have build time configurations (for example, build without FrSky telemetry), then this list will be very short, but for now all this libraries are the ones needed to do a build - UAVCAN isn't though, but as I said, let's leave it for now.